### PR TITLE
Migrate to slog in LSP

### PR DESCRIPTION
**Pull request by shuheiktgw**
## NOTE

SSIA. Use slog instead of the old plain logger.

### CODE REVIEW

File: tools/go.mod [hunk 1]
```diff
@@ -5,29 +5,29 @@ go 1.21
 require (
 	github.com/bufbuild/buf v1.21.0
 	github.com/envoyproxy/protoc-gen-validate v1.0.2
-	github.com/golangci/golangci-lint v1.53.3
+	github.com/golangci/golangci-lint v1.54.2
```
Comment (shuheiktgw) [inline tools/go.mod:8]: I need to update the linter to support log/slog for gci

**Comment by goccy** [comment]
Probably, I think we need to run `go mod tidy` at repository root.

**Comment by shuheiktgw** [comment]
Miss that! Fixed it 🙇

**Review by goccy — APPROVED** [review]

**Comment by goccy** [comment]
LGTM !

**Pull request merged**